common/src/main/java/me/lucko/luckperms/common/commands/misc/ExportCommand.java

```diff
@@ -68,7 +71,16 @@ public void execute(LuckPermsPlugin plugin, Sender sender, ArgumentList args, St
             exporter = new Exporter.WebUpload(plugin, sender, includeUsers, includeGroups, label);
         } else {
             Path dataDirectory = plugin.getBootstrap().getDataDirectory();
-            Path path = dataDirectory.resolve(args.get(0) + ".json.gz");
+            Path path;
```

Comment:
Initializing the path variable here to what’s on line 82 would remove the need for the else statement

Reply:
Won't work, `args.get(0)` will throw index out of bounds